spec.md

```diff
@@ -99,7 +97,8 @@ The `Span` interface must have the following capabilities:
 
 The `Tracer` interface must have the following capabilities:
 
-- Start a `Span` that has no parent, commonly referred to as a *root* `Span` **(py: `start_trace`, go: `StartTrace`)**
+- Start a new child `Span` with a given operation name. With no parent, commonly referred to as a *root* `Span`, no trace attributes exist. With a parent any trace attributes must be passed through into the created child. **(py: `create_span`, go: `CreateSpan`)**
```

Comment:
One other benefit of having `start_trace` handle both start + join but be separate from `create_span`: this may be important for a tracer to understand throughput in a system.  If the instrumenter is manually guarding trace points, the Tracer will not be able to know true throughput.